core/server/services/members/config.js

```diff
@@ -154,6 +154,10 @@ class MembersConfigProvider {
         };
     }
 
+    getEnableStripePromoCodes() {
+        return this._config.get('enableStripePromoCodes');
```

Comment:
Is this an OK way to accept that config input? Just wondering if there's a place where nconf validates this as a boolean, or if I should just do something like `this._config.get('enableStripePromoCodes') === true` instead

Reply:
This looks good to me :relaxed: